extensions/interactions/EndExploration/directives/EndExplorationValidationService.js

```diff
@@ -21,10 +21,21 @@ oppia.factory('EndExplorationValidationService', [
   function(WARNING_TYPES, baseInteractionValidationService) {
     return {
       getCustomizationArgsWarnings: function(customizationArgs) {
+        var warningsList = [];
         baseInteractionValidationService.requireCustomizationArguments(
           customizationArgs, ['recommendedExplorationIds']);
-        // TODO(juansaba): Implement customization args validations.
-        return [];
+
+        var recommendedExplorationIds = (
+            customizationArgs.recommendedExplorationIds.value);
+        for (var i = 0; i < recommendedExplorationIds.length; i++) {
+          if (!angular.isString(recommendedExplorationIds[i])) {
+            warningsList.push({
+              type: WARNING_TYPES.ERROR,
+              message: 'Recommended Exploration ID must be a string.'
```

Comment:
`Each recommended exploration ID must be a string.`

(No need to capitalize "Exploration".)

Reply:
Done!